tests/10-migration-backend-tests/verification/verify_migrations_test.go

```diff
@@ -186,7 +249,7 @@ func Test_VerifyTaxonomyCopyrightAndUse(t *testing.T) {
 
 	// retrieve json of the migrated entity from the jsonapi and unmarshal the single response
 	copyrightRes := &JsonApiCopyrightAndUse{}
-	u.getSingle(copyrightRes)
+	u.get(copyrightRes)
```

Comment:
Did this get changed from `getSingle` because there are multiple copyright and use entries being returned?  If that isn't intended behavior then we'd need to investigate why.  If it is intended behavior, I'm not sure how the access terms implementation of migration affects this test.

Reply:
Hmmm... I don't believe I meant to do that, or why I'd even touch that function. I'll revert that change.

Reply:
Thanks for catching this.